### DISCUSSION TITLE
ui exception while fetching data

### DISCUSSION THREAD

**Discussion by houbinglei** [apache/skywalking#10014] [Q&A]
when i open the skywalking ui, i can see the page like this exception while fetching data; but when i refresh the page , it will be ok... It's strange that the exception page comes sometimes.
![图片](https://user-images.githubusercontent.com/37292926/203515416-5e17c9be-a669-42d4-8ae4-5398acd2b993.png)

how can i resolve this?

**Comment by wu-sheng** [comment]
This usually means you accessed the UI before backend is ready.

**Comment by houbinglei** [comment]
thanks! 
the backend means tha java app itself?  it is error randomly。but  how can i resolve this?

**Reply by Superskyyy** [reply]
Upgrade OAP to latest and should be all good.

**Reply by houbinglei** [reply]
thanks! 
i have upgrade oap to the latest;
but the data collected from als is missing......

![image](https://user-images.githubusercontent.com/37292926/216249087-2ee63ae5-2bdb-4fd0-91ef-7f4ab6cda610.png)

could you give me some advice...

**Reply by wu-sheng** [reply]
The service itself is from ALS, if you open service mesh dashboard.
Make sure you chose the correct time range, which range has traffic.

**Reply by houbinglei** [reply]
thanks for your help! i will check it !
